tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -61,19 +88,25 @@ const Calendar = () => {
 export default function CalendarScreen ({ navigation }) {
     const [dropdownExpanded, setDropdownExpanded] = useState(false);
     const [selectedView, setSelectedView] = useState(VIEWS.Nothing);
+    const toggleSelectedView = (targetView) => {
+        if (selectedView === targetView){
+            setSelectedView(VIEWS.Nothing);
+        }
+        else {
+            setSelectedView(targetView);
+        }
+    }
     const renderedArrow = dropdownExpanded ? <Icon name="keyboard-arrow-up" size={24}/> : <Icon name="keyboard-arrow-down" size={24} />
     return (
         <View style={styles.container}>
             <View style={styles.navbarContainer}>
-                <View style={styles.backButtonContainer}>
-                    <BackButton
-                        onPress={() => {
-                            navigation.navigate('Year')
-                        }}
-                        title='Year'
-                        width={sideComponentWidth}
-                    />
-                </View>
+                {/*<BackButton
+                    onPress={() => {
+                        navigation.navigate('Year')
+                    }}
+                    title='Year'
+                    width={sideComponentWidth}
+                />*/}
```

Comment:
this reminds me we need to get rid of the unused blank year page lol - but we'll do that in another story 😅